src/main/groovy/com/google/appengine/AppEnginePlugin.groovy

```diff
@@ -245,28 +256,32 @@ class AppEnginePlugin implements Plugin<Project> {
         }
     }
 
-    private void configureAppConfig(Project project, AppEnginePluginExtension appEnginePluginExtension) {
+    private void configureWorkingDirectory(Project project) {
+        AppConfigTaskTemplate.metaClass.getWorkingDirectory = { -> this.explodedAppDirectory }
```

Comment:
In regard to `workingDirectory`, I agree, looks ugly. What do you think about DSL config flag that will point which directory to use, like:
```
appengine {
    useOutputWebappDir=true
}
```
instead of system property "use.output.webapp.dir"?
Probably, it doesn't reflect the actual meaning, requires documentation, or may be different name.